src/operations/identify.rs

```diff
@@ -135,12 +186,64 @@ mod tests {
                     color_type: ExtendedColorType::Cmyk8,
                 },
             },
+            IdentifyFormat { template: None },
             &mut output,
         )
         .unwrap();
         assert_eq!(
             String::try_from(output).unwrap(),
-            "image_without_format.jpg 1x1 1x1+0+0 8-bit CMYK\n"
+            // TODO: Should the missing image format turn into a space or not?
```

Comment:
since the test enforces a behavior that we don't know is correct or not, let's just remove it